### PR TITLE
#3 Fix for the missing `PlatformSettings__ApiAuthorizationEndpoint`  and removed unnecessary elements

### PR DESCRIPTION
**Pull request by codemascot**
Fix for the missing `PlatformSettings__ApiAuthorizationEndpoint` and removed unnecessary elements

## Description
1. Fix for the missing `PlatformSettings__ApiAuthorizationEndpoint`
2. Removed unnecessary elements

## Related Issue(s)
- #3 
- #360 

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] All tests run green

### CODE REVIEW

File: src/Storage/Configuration/RegisterServiceSettings.cs
Comment (SandGrainOne) [inline src/Storage/Configuration/RegisterServiceSettings.cs:7]: We must also change the name of the environment variables in the ops project.

**Review by SandGrainOne — APPROVED** [review]

File: src/Storage/Configuration/RegisterServiceSettings.cs
Comment (codemascot) [reply src/Storage/Configuration/RegisterServiceSettings.cs:7]: You are correct! 💡 👍 Was waiting for the approval of this PR. Now will have a look at the environment variable in the ops project! 🙂

**Pull request merged**
